### PR TITLE
Development

### LINKED ISSUE #25: [Question] AnythingLLM Local Docker installation guide
On https://github.com/Mintplex-Labs/anything-llm/blob/master/docker/HOW_TO_USE_DOCKER.md for Windows install guide 3rd line has ` -ItemType File`
image

But on https://docs.useanything.com/getting-started/installation/self-hosted/local-docker for Windows install guide 3rd line don't have ` -ItemType File`
image

### Which one is correct? the code on Github repo or Docs?
Note: I noticed our docs is missing `docker pull mintplexlabs/anythingllm` command and some more info so I can fix all of these on my next PR

### PR DESCRIPTION
**Pull request by ShadowArcanist**
### Pull Request Type

- [x] ✨ feat

### Relevant Issues

resolves #25 

### What is in this change?

**✨ Added Prettier for code consistency**

**🔄 Updated Local docker install guide**
- Added the missing `docker pull` command
- Fixed the missing word on the windows installation guide mentioned on #25 

**🔄 Updated macOS install guide**
- Added Installation guide using Homebrew

**⬆ Bumped release version on Banner + FAQ title update**
- Bumped the latest version number on Banner from 1.5.4 > 1.5.5
- Updated the title of `LLM not using docs` FAQ

### Validations

- [x] Ensured updated documentation pass spell check
- [x] Updated or added relevant links as needed
- [x] Reviewed the changes for clarity and accuracy
- [x] Successfully ran the code locally without encountering errors

### CODE REVIEW

File: pages/faq/llm-not-using-my-docs.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Why does the LLM not use my documents"
+title: "Why does the LLM is not using my documents"
```
Comment (timothycarambat) [inline pages/faq/llm-not-using-my-docs.mdx:2]: The correct language here is
`Why is the LLM not using my documents`. The original text was okay as well and did not need to be changed

File: pages/getting-started/installation/desktop/macos.mdx [hunk 2]
```diff
@@ -94,21 +101,30 @@ Once the dmg opens, you can drag the AnythingLLM logo into `Applications`
 
 Once installed, you will find AnythingLLM in your Applications folder as well as you can use `cmd + spacebar` and type in AnythingLLM to run.
 
+---
+
+## Install using Homebrew
```
Comment (timothycarambat) [inline pages/getting-started/installation/desktop/macos.mdx:106]: TIL, i had not idea someone did this for us

File: theme.config.tsx [hunk 3]
```diff
@@ -93,10 +93,10 @@ const config: DocsThemeConfig = {
   ),
   banner: {
     dismissible: true,
-    key: 'v1.5.4-release', // Storage key to keep the banner state (dismissed or not). If you have updated your banner text, you should change the key to make sure the banner is shown again.
+    key: 'v1.5.5-release', // Storage key to keep the banner state (dismissed or not). If you have updated your banner text, you should change the key to make sure the banner is shown again.
     text: (
       <a href="https://useanything.com/download" target="_blank">
-        🚀 AnythingLLM v1.5.4 is out. Update now →
+        🚀 AnythingLLM v1.5.5 is out. Update now →
```
Comment (timothycarambat) [inline theme.config.tsx:99]: Ah, i forgot this!

**Review by timothycarambat — CHANGES_REQUESTED** [review]
Can we revert the text change for the `"Why does the LLM not use my documents"` change. The modifications made are not grammatically correct now, it was okay as it was. everything else looks very good and nice to have prettier support for this now

File: pages/faq/llm-not-using-my-docs.mdx (hunk 1 above)
Comment (ShadowArcanist) [reply pages/faq/llm-not-using-my-docs.mdx]: Ahh okay! I reverted the change back to the original (:

**Review by timothycarambat — APPROVED** [review]

**Pull request merged**